website/pages/use-cases/secrets-management/content.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
I took the `content.json` approach on this page because it felt like the best fit (other pages didn't feel like they had enough content yet to warrant a separate content file).